platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/sharing_build_logic_between_subprojects.adoc

```diff
@@ -12,354 +12,223 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
 [[sharing_build_logic_between_subprojects]]
-= Sharing Build Logic between Subprojects
+= Sharing Build Logic using `buildSrc`
 :keywords: buildsrc,build-logic
 
-Subprojects in a multi-project build typically share some common dependencies.
-
-image::structuring-builds-3.png[]
-
-Instead of copying and pasting the same Java version and libraries in each subproject build script, Gradle provides a special directory for storing shared build logic that can be automatically applied to subprojects.
-
-[[sec:using_buildsrc]]
-== Share logic in `buildSrc`
-
-`buildSrc` is a Gradle-recognized and protected directory which comes with some benefits:
-
-1. *Reusable Build Logic*:
-+
-`buildSrc` allows you to organize and centralize your custom build logic, tasks, and plugins in a structured manner.
-The code written in buildSrc can be reused across your project, making it easier to maintain and share common build functionality.
+Subprojects in a multi-project build often share common dependencies.
 
-2. *Isolation from the Main Build*:
-+
-Code placed in `buildSrc` is isolated from the other build scripts of your project.
-This helps keep the main build scripts cleaner and more focused on project-specific configurations.
+Rather than duplicating the same dependency declarations across multiple build scripts, Gradle allows you to centralize shared build logic in a special directory.
+This way, you can declare the dependency version in one place and have it automatically apply to all subprojects.
 
-3. *Automatic Compilation and Classpath*:
-+
-The contents of the `buildSrc` directory are automatically compiled and included in the classpath of your main build.
-This means that classes and plugins defined in buildSrc can be directly used in your project's build scripts without any additional configuration.
+image::structuring-builds-8.png[]
 
-4. *Ease of Testing*:
-+
-Since `buildSrc` is a separate build, it allows for easy testing of your custom build logic.
-You can write tests for your build code, ensuring that it behaves as expected.
-
-5. *Gradle Plugin Development*:
-+
-If you are developing custom Gradle plugins for your project, `buildSrc` is a convenient place to house the plugin code.
-This makes the plugins easily accessible within your project.
-
-The `buildSrc` directory is treated as an <<composite_builds.adoc#composite_build_intro,included build>>.
+[[sec:sharing_logic_via_convention_plugins]]
+== Using `buildSrc` as a convention plugin
```

Comment:
Wondering about this title, maybe "Using `buildSrc` for convention plugins"? buildSrc itself is not a convention plugin, it provides a location to put convention plugins

Reply:
I also hated the title and couldn't think of a better one. I've shortened it to "Using `buildSrc`".